### ISSUE TITLE
Error: package or namespace load failed for ‘sjPlot’

### ISSUE THREAD

**Issue by anaveda** [strengejacke/sjPlot#346]
Hi, I am getting this error when trying to load the package. This came after upgrading other packages in R-Studio. I uninstall and re install again the package but the same error continue. Thanks

**Comment by strengejacke** [comment]
Can you post the complete error message? I guess you have not installed all necessary packages.

**Comment by anaveda** [comment]
Sure,
Is this
> library(sjPlot)
Error: package or namespace load failed for ‘sjPlot’:
 object ‘get_note’ is not exported by 'namespace:sjlabelled'
In addition: Warning message:
package ‘sjPlot’ was built under R version 3.4.4 

I installed in other computer will all dependencies and got the same message

**Comment by strengejacke** [comment]
Ok, please update _sjlabelled_ from CRAN. Looks like you have installed the dev-version from GitHub.

**Comment by anaveda** [comment]
Hi Daniel,
It didnt work. Now I get this
Error: package or namespace load failed for ‘sjPlot’:
 object ‘get_note’ is not exported by 'namespace:sjlabelled'
In addition: Warning messages:
1: package ‘sjPlot’ was built under R version 3.4.4 
2: In checkMatrixPackageVersion() :
  Package version inconsistency detected.
TMB was built with Matrix version 1.2.14
Current Matrix version is 1.2.12
Please re-install 'TMB' from source using install.packages('TMB', type = 'source') or ask CRAN for a binary version of 'TMB' matching CRAN's 'Matrix' package

**Comment by strengejacke** [comment]
Can you post your `sessionInfo()`? You have to make sure you have either version 1.0.8 or 1.0.9 from sjlabelled.

**Comment by anaveda** [comment]
This is the sessionInfo. And yes, the version of sjlabelled installed is 1.09
> sessionInfo()
R version 3.4.3 (2017-11-30)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] lmerTest_3.0-1 lme4_1.1-17    Matrix_1.2-12 

loaded via a namespace (and not attached):
 [1] tidyr_0.8.0         splines_3.4.3       merTools_0.3.0      carData_3.0-1       modelr_0.1.1       
 [6] shiny_1.0.5         assertthat_0.2.0    stats4_3.4.3        coin_1.2-2          numDeriv_2016.8-1  
[11] pillar_1.2.1        lattice_0.20-35     glue_1.2.0          digest_0.6.15       promises_1.0.1     
[16] glmmTMB_0.2.0       snakecase_0.9.1     minqa_1.2.4         colorspace_1.3-2    sandwich_2.4-0     
[21] httpuv_1.4.1        htmltools_0.3.6     survey_3.33-2       plyr_1.8.4          psych_1.8.3.3      
[26] pkgconfig_2.0.1     broom_0.4.4         haven_1.1.1         purrr_0.2.4         xtable_1.8-2       
[31] mvtnorm_1.0-7       scales_0.5.0        stringdist_0.9.4.7  later_0.7.1         emmeans_1.1.3      
[36] arm_1.10-1          tibble_1.4.2        effects_4.0-1       bayesplot_1.5.0     ggplot2_2.2.1      
[41] DT_0.4              sjlabelled_1.0.9    TH.data_1.0-8       TMB_1.7.13          nnet_7.3-12        
[46] lazyeval_0.2.1      cli_1.0.0           mnormt_1.5-5        mime_0.5            survival_2.41-3    
[51] magrittr_1.5        crayon_1.3.4        estimability_1.3    nlme_3.1-131        MASS_7.3-47        
[56] forcats_0.3.0       foreign_0.8-69      tools_3.4.3         data.table_1.10.4-3 multcomp_1.4-8     
[61] stringr_1.3.0       munsell_0.4.3       prediction_0.3.2    bindrcpp_0.2.2      ggeffects_0.3.2    
[66] compiler_3.4.3      rlang_0.2.0         blme_1.0-4          grid_3.4.3          nloptr_1.0.4       
[71] ggridges_0.5.0      rstudioapi_0.7      htmlwidgets_1.2     gtable_0.2.0        codetools_0.2-15   
[76] abind_1.4-5         sjstats_0.14.2-3    reshape2_1.4.3      sjmisc_2.7.1        R6_2.2.2           
[81] zoo_1.8-1           knitr_1.20          pwr_1.2-2           dplyr_0.7.4         bindr_0.1.1        
[86] modeltools_0.2-21   stringi_1.1.7       parallel_3.4.3      Rcpp_0.12.16        lmtest_0.9-36      
[91] tidyselect_0.2.4    coda_0.19-1

**Comment by strengejacke** [comment]
You are right - I forgot to add that function to the namespace. That's why the check for Debian complained, however, I wonder why all other checks, including on CRAN, were ok.

As a workaround, please downgrade to sjlabelled 1.0.8
